plugins/org.python.pydev/src/org/python/pydev/ui/pythonpathconf/package_manager/CondaPackageManager.java

```diff
@@ -35,6 +37,21 @@ public CondaPackageManager(IInterpreterInfo interpreterInfo, File prefix) {
         this.prefix = prefix;
     }
 
+    public static List<String> listCondaEnvironments(File condaExecutable) {
+        List<String> lst = new ArrayList<>();
+        String encoding = Charset.defaultCharset().displayName(); // use system encoding
```

Comment:
To use the system encoding, just passing `null` should work, but since we're now dealing with json, I actually expect it to use `utf-8` (so, please change to `encoding = "utf-8")`.